pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -133,7 +132,7 @@ func (o *OAuthAPIServerOptions) NewOAuthAPIServerConfig() (*apiserver.Config, er
 	}
 
 	serverConfig.GenericConfig.OpenAPIConfig = openapiconfig.DefaultOpenAPIConfig()
-	serverConfig.GenericConfig.OpenAPIV3Config = openapiconfig.DefaultOpenAPIConfig()
+	serverConfig.GenericConfig.OpenAPIV3Config = openapiconfig.DefaultOpenAPIV3Config()
```

Comment:
why the change ?

Reply:
`DefaultOpenAPIConfig()` returns `*common.Config` which is incompatible with `common.OpenAPIV3config` (the type of `serverConfig.GenericConfig.OpenAPIV3Config`).